activerecord/test/cases/job_runtime_test.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
```

Comment:
This entire file should be in the `activerecord/test/activejob`

Reply:
@rafaelfranca Done!